src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -118,15 +121,19 @@ private String datasetName() {
     // datasets from the DAO at the end of a test, you'll see a FK violation when `connectedOperations`
     // tries to delete the resource profile generated in `setup()`.
     private Dataset readDataset(String requestFile) throws Exception {
+        String datasetName = makeDatasetName();
         DatasetRequestModel datasetRequest = jsonLoader.loadObject(requestFile, DatasetRequestModel.class);
         datasetRequest
             .defaultProfileId(profileModel.getId())
-            .name(datasetName());
+            .name(datasetName);
+        UUID projectId = resourceService.getOrCreateDatasetProject(datasetName, profileModel);
         Dataset dataset = DatasetUtils.convertRequestWithGeneratedNames(datasetRequest);
+        dataset.projectResourceId(projectId);
```

Comment:
Any reason why these are not chained calls? In other places the code seems to prefer chaining these types of calls.

Reply:
I don't think we have made a strong preference, but I like the chaining. I'll change it.